addok_france/utils.py

```diff
@@ -126,7 +126,7 @@ def fold_ordinal(s):
 
 def remove_leading_zeros(s):
     """0003 => 3."""
```

Comment:
Humm, I'd say the spirit of the function hasn't changed, so I'd keep the docstring focused on it. And let unittests monitor implementation details.
Thoughts?